dockers/docker-dhcp-relay/cli/show/plugins/dhcp_relay.py

```diff
@@ -0,0 +1,22 @@
+from natsort import natsorted
+from .. import vlan
+
+
+def get_dhcp_helper_address(ctx, vlan):
+    cfg, _ = ctx
+    vlan_dhcp_helper_data, _, _ = cfg
+    vlan_config = vlan_dhcp_helper_data.get(vlan)
+    if not vlan_config:
+        return ""
+
+    dhcp_helpers = vlan_config.get('dhcp_servers', [])
+
+    return '\n'.join(natsorted(dhcp_helpers))
+
+
+vlan.VlanBrief.COLUMNS.append(('DHCP Helper Address', get_dhcp_helper_address))
```

Comment:
Would this be problematic, plugins changing parent data structures ?  There probably needs to be better mechanisms for this ?

Reply:
Would it be better if VlanBrief had a public class method:
```python
def register_column(column_name, column_data_callback)
```
And a comment that this API is in use by plugins to register additional columns?